docs/topics/commonly_used_plugins.rst

```diff
@@ -10,284 +10,69 @@ Some commonly-used plugins
     you are upgrading from a previous version. Please refer to
     :ref:`Upgrading from previous versions <upgrade-to-3.0>`
 
-These are the recommended plugins to use with django CMS.
 
-..  module:: djangocms_file
+Please note that dozens if not hundreds of different django CMS plugins have been made made available under open-source
+licences. Some, like the ones on this page, are like to be of general interest, while others are highly specialised.
 
-..  class:: djangocms_file.cms_plugins.FilePlugin
+This page only lists those that fall under the responsibility of the django CMS project. Please see the `Django Packages <https://djangopackages.org/search/?q=django+cms>`_ site for some more, or just do a web search for the functionality
```

Comment:
I think definitely the project, not the core team. They fall under the auspices of the project as a whole.